manifest.common.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 3,
   "name": "Hoppscotch Browser Extension",
-  "version": "0.37",
+  "version": "0.38",
```

Comment:
The latest version is still at `0.36` and the version bump might not be necessary.

Reply:
reverted.